README.md

```diff
@@ -68,5 +68,13 @@ defmodule MyApplicationTest do
 end
 ```
 
+### Shared Mode
+If you want to inject a dependency that will be shared by all processes, you can do so by passing the `shared: true` option.
+This can be useful if you have background processing. This is only recommended for tests that do not run async.
+
+```elixir
+register(File, FileStub, shared: true)
+```
+
 ## TODO
 - Add `allow/1` for enabling another process to use registrations from the current test.
```

Comment:
is this the same thing here or something else?

Reply:
`allow/1` would work differently.  It would work like `SqlSandbox.allow` works where it would allow the specific process you pass to access the registrations, whereas shared mode just gives every pid access.